packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.test.tsx

```diff
@@ -18,6 +19,33 @@ describe('ChoiceGroupOption', () => {
         <ChoiceGroupOption iconProps={{ iconName: 'CalendarWeek' }} key="G" text="Option G" focused />
         <ChoiceGroupOption iconProps={{ iconName: 'CalendarDay' }} key="H" text="Option H" checked />
         <ChoiceGroupOption iconProps={{ iconName: 'CalendarWeek' }} key="I" text="Option I" disabled />
+        <ChoiceGroupOption
+          key="J"
+          text="Option J"
+          onRenderLabel={(option: IChoiceGroupOptionProps): JSX.Element => {
```

Comment:
To get the build to pass you'll need to add a comment line `// tslint:disable:jsx-no-lambda` (either around here or near the top of the file). That lint rule generally doesn't matter for tests, so disabling it is fine.